main.go

```diff
@@ -338,3 +359,13 @@ func parseCsv(value string) map[string]string {
 
 	return tags
 }
+
+func parseCopyLabels(copyLabelsString string) []string {
+	if copyLabelsString == "*" {
+		return []string{"*"}
+	}
+	if copyLabelsString == "" {
+		return []string{}
+	}
+	return strings.Split(copyLabelsString, ",")
```

Comment:
what if the user enters `foo,*,bar`?

Reply:
Briefly thought about it, but did not have a solution other than "treat a single `*` entry as special" and document that is how to enable wildcard mode.

In the case of `foo,*,bar` should we enable wildcard mode? I could see someone making a typo while trying to do `foo*,bar` (btw `foo*` won't work) by adding a comma and then having their more specific label keys ignored suddenly all label keys are copied, which might be a surprising outcome.

We originally thought we would use wildcard mode but find ourselves using a more specific allow-list instead (though in my testing wildcard mode looked fine for general use). We could drop support for the magic `*`. And either add a new bool `--copy-all-labels` now, or not support labels at all and if a compelling request comes up then add the new bool flag.

Reply:
I think the simplest solution is to just check for the string either being exclusively `*` or not in it at all.